pkg/util/export/etl/db/db_holder.go

```diff
@@ -181,235 +150,131 @@ func WriteRowRecords(records [][]string, tbl *table.Table, timeout time.Duration
 	ctx, cancel := context.WithTimeout(context.Background(), timeout)
 	defer cancel()
 
-	err = bulkInsert(ctx, dbConn, records, tbl, MaxInsertLen, MiddleInsertLen)
+	err = bulkInsert(ctx, dbConn, records, tbl)
 	if err != nil {
+		fmt.Printf("gavinyue bulkInsert failed, err: %v\n", err)
```

Comment:
It seems that the use of fmt.Printf to print information is not allowed

Reply:
removed